Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -1812,7 +1812,7 @@ void PreferencesDlg::saveCurrentMessage()
         QString paramKey = eventInfo.settingKey;
         QString text = ui.TTSMsgEdit->text();
 
-        if (!text.isEmpty() && text != UtilTTS::getDefaultValue(paramKey) && text != ttSettings->value(paramKey))
+        if (!text.isEmpty() && text != ttSettings->value(paramKey))

```

Comment:
shouldn't `ttSettings` clear the value if `text == UtilTTS::getDefaultValue(paramKey)` ?

So instead:
```
if (!text.isEmpty())
{
    ttSettings->setValue(paramKey, text == UtilTTS::getDefaultValue(paramKey) ? "" : text);
}
```